Source/WebKit/UIProcess/ios/WKPDFView.mm

```diff
@@ -199,6 +208,10 @@ - (instancetype)web_initWithFrame:(CGRect)frame webView:(WKWebView *)webView mim
     _keyboardScrollingAnimator = adoptNS([[WKKeyboardScrollViewAnimator alloc] initWithScrollView:webView._scrollViewInternal]);
     _webView = webView;
 
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
+    [[_webView _wkScrollView] addObserver:self forKeyPath:@"contentSize" options:NSKeyValueObservingOptionNew context:kvoContext];
```

Comment:
I wonder if it's sufficient to override `setContentSize:` rather than use KVO?

Reply:
(same question)